extensions/emmet/extension.toml

```diff
@@ -0,0 +1,11 @@
+id = "emmet"
+name = "Emmet"
+description = "Emmet support"
+version = "0.0.1"
+schema_version = 1
+authors = []
+repository = "https://github.com/zed-extensions/emmet"
```

Comment:
It typo?
```suggestion
repository = "https://github.com/zed-extensions/zed"
```

Reply:
AFAIK this should point to the repository for that particular extension so I don't think it's a typo.

Reply:
We’ll want the repository URL to be the Zed repo’s URL, for now. When this does get extracted to its own repo we can update the repository URL.

Although now I’m wondering if we should create this extension under the zed-extensions org to begin with, since it’s not being extracted from a built-in language.